src/main/lib/exclusion-utils.js

```diff
@@ -0,0 +1,35 @@
+function isDirExclusion (exclusion) {
+  return exclusion.indexOf('.') === -1;
+}
+
+function isExcluded (filePath, exclusion) {
```

Comment:
Why did you implement your own exclusion logic instead of relaying on `matcher` library as it was done before? This could be simpler IMO.

Reply:
Before as we were dealing with the fs Dirent objects we could access the name of the current directory/file, now using glob we lose that information so the exclusion logic becomes a bit more complex

Reply:
However I could check if the exclusion is appears to be a file or a directory and wrap them in '*' appropriately which may clean up some of the logic

Reply:
No longer need to have own exclusion logic, mocked old logic instead